### PR TITLE
RealtimeKafkaToStream example

### PR DESCRIPTION
**Pull request by jwang47**

### CODE REVIEW

File: RealtimeKafkaToStream/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Realtime Kafka To Stream Adapter Configuration
```
Comment (gokulavasan) [inline RealtimeKafkaToStream/README.rst:1]: I thought we were doing Realtime Kafka To HBase

Comment (jwang47) [inline RealtimeKafkaToStream/README.rst:1]: Oops, will fix.

File: RealtimeKafkaToStream/README.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach).
+
+Note: You need to fill in the ``kafka.zookeeper`` configuration in config.json before creating the Adapter.
+This is the connection string to your Kafka server in ZooKeeper.
+
+::
+
+  cdap> create adapter kafkaIngest RealtimeTwitterToHBase/config.json
+  Successfully created adapter 'KafkaIngest'
```
Comment (bdmogal) [inline RealtimeKafkaToStream/README.rst:21]: 'K' should be lowercase

File: RealtimeKafkaToStream/README.rst (hunk 1 above)
Comment (jwang47) [inline RealtimeKafkaToStream/README.rst:1]: Reopened PR https://github.com/cdap-guides/cdap-etl-adapter-guide/pull/3

**Pull request closed without merging**